pkg/authorization/api/synthetic.go

```diff
@@ -3,6 +3,7 @@ package api
 // Synthetic authorization endpoints
 const (
 	DockerBuildResource          = "builds/docker"
+	OptimizedDockerBuildResource = "builds/optimizeddocker"
```

Comment:
In the future we're just going to take this away.  Nothing about the current path is less secure than Docker, it's just a hedge so we can selective enable it.